llap-server/src/java/org/apache/hadoop/hive/llap/daemon/impl/ContainerRunnerImpl.java

```diff
@@ -280,8 +276,12 @@ public SubmitWorkResponseProto submitWork(SubmitWorkRequestProto request) throws
       Configuration callableConf = new Configuration(getConfig());
       UserGroupInformation fsTaskUgi = fsUgiFactory == null ? null : fsUgiFactory.createUgi();
       boolean isGuaranteed = request.hasIsGuaranteed() && request.getIsGuaranteed();
+
+      boolean addTaskTimes = callableConf.getBoolean(ConfVars.TEZ_EXEC_SUMMARY.varname, false)
```

Comment:
i don't like that we have a duplication of default values here and in the actual hive config can we avoid this ?

Reply:
Done